wayback-core/src/main/java/org/archive/wayback/exception/RangeNotSatisfiable.java

```diff
@@ -0,0 +1,38 @@
+package org.archive.wayback.exception;
+
+import javax.servlet.http.HttpServletResponse;
+
+import org.archive.wayback.core.Resource;
+
+/**
+ * RangeNotSatisfiable is thrown when selected Resource does not have content data
+ * that can satisfy requested range.
+ * This happens when the Resource itself is a capture of range request that does not cover
+ * requested range.
+ * @see org.archive.wayback.replay.TransparentReplayRenderer
+ */
+public class RangeNotSatisfiable extends SpecificCaptureReplayException {
```

Comment:
That's fine, too.
As you might guess, I named it after the standard name for 406 response "Requested Range Not Satisfiable". It is for returning 406 response, and it is not meant to be caught by that specific name.  So I didn't see a need to follow that -Exception convention.